app/notifications/reimbursement_complete_notification.rb

```diff
@@ -0,0 +1,19 @@
+class ReimbursementCompleteNotification < BaseNotification
+  deliver_by :database
+
+  param :case_contact
+
+  def title
+    "Reimbursement Approved"
+  end
+
+  def message
+    case_contact = params[:case_contact]
+    # Questions about logic for payment rate
+    "Volunteer #{case_contact.creator.display_name}'s request for reimbursement for #{case_contact.miles_driven}mi on #{case_contact.occurred_at_display} has been processed and is en route."
```

Comment:
I think it's supposed to include the total based on the [mileage rate](https://github.com/rubyforgood/casa/blob/main/app/models/mileage_rate.rb) When a mileage rate goes into effect all miles are reimbursed for the amount until a mileage rate with a later date goes into effect. 

In the part where the ticket says "for $12"

There could be no mileage rate.

Reply:
I added some logic to set this if the casa_org has active mileage rates. Can see the main logic in CasaOrg.mileage_rate_for_given_date(DateTime)